code/src/assets/pages.json

```diff
@@ -29,7 +29,11 @@
     "title":"Countries",
     "description":"",
     "alias":"countries"
+    },
+    {
+    "page_id":"5",
+    "title":"My Profile",
+    "description":"",
+    "alias":"myprofile"
     }
```

Comment:
@nilesh-more   Please remove this object as we do not need it as of now

Reply:
@sushan05 resolved